bat-utils/lib/runtime-wallet.js

```diff
@@ -346,8 +351,18 @@ Wallet.providers.uphold = {
     }
 
     const altScale = this.currency.alt2scale(info.altcurrency)
-    const balanceProbi = new BigNumber(cardInfo.balance || 0).times(altScale)
-    const spendableProbi = new BigNumber(cardInfo.available || 0).times(altScale)
+    const { error } = Joi.validate(cardInfo, cardInfoSchema)
+    if (error) {
+      this.runtime.captureException(error, {
```

Comment:
would it be more useful to check this inside bat-utils where we have more context?

Reply:
Sorry, i don't understand what more context we would have. I'm happy to, I just don't understand the reason.

Reply:
we have the full response from uphold

Reply:
yeah, we have the response in this function. so that means we have more context, no?

Reply:
i'm blind :)